### ISSUE TITLE
Docs on using Cypress in a devcontainer

### ISSUE THREAD

**Issue by johnnyreilly** [cypress-io/cypress#7884]
Hello everyone!

I ❤️ Cypress - it is an *amazing* product.  Thanks for your work on it. 

I've found myself doing more and more work lately using [devcontainers](https://code.visualstudio.com/docs/remote/containers) as my development environment in VS Code.

The one thing that I haven't yet cracked is working out how to get Cypress working in a devcontainer.  I've been able to get a part of the way there by using the Node example devcontainer here: https://github.com/microsoft/vscode-dev-containers/tree/master/containers/javascript-node-14

And tweaking it based upon the Cypress docs for CI here: https://docs.cypress.io/guides/guides/continuous-integration.html#Dependencies to add in the Cypress dependencies.

The tricky part is that whilst this gets us to the point of Cypress technically running, it doesn't allow for the Cypress Test Runner that you typically use with `cypress open .`.  https://docs.cypress.io/guides/guides/command-line.html#cypress-open to fire up.

I'm not sure what the way forward here would look like and I'm curious if there are docs that explain how to get up and running with devcontainers.  I haven't found any and I suspect this is going to become an increasingly common workflow so I thought I'd ask!

**Comment by jennifer-shehane** [comment]
@johnnyreilly Yes, Cypress requires Electron. There's an issue to separate this dependency from Cypress here: https://github.com/cypress-io/cypress/issues/3899 But yeah, there is likely some base dependencies missing in the devcontainers that make Cypress not open. 

We have docker-images https://github.com/cypress-io/cypress-docker-images that may be helpful in tracking down whats missing. 

This repo is only for issues relating to the Cypress product. If you want to open an issue on our docs, you can open it [here](https://github.com/cypress-io/cypress-documentation/issues/new). This issue will be moved to that repository.
